org/wildfly/remoting/http-listerner-registry/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.remoting.http-listener-registry
+
+DESCRIPTION: Provides a registry of all Undertow listeners, and the services that are registered on them.
+
+REGISTERED BY:
+  
+  NAME: WildFly Core
+  URL:  https://github.com/wildfly/wildfly-core
+
+DYNAMIC: false
+
+PRIVATE: false
+
+SUPPORTS RUNTIME ONLY: true
```

Comment:
This must be false. A dynamic capability cannot be a runtime-only requirement because the dynamic part of the required capability's name must come from configuration, and that means it's not a runtime-only requirement.

See also "Supporting runtime-only requirements" in https://github.com/wildfly/wildfly-capabilities/blob/master/README.md

Reply:
Sorry; I misread or something -- it says DYNAMIC: false

As we just chatted about the other factor to consider is whether this capability needs to support 'true' and might ever want to switch to 'false' (e.g. the providing resource might no longer require a reload to be removed.)  If we say 'true' it's a permanent part of the contract.

Reply:
@bstansberry I've relaxed the requirement, I'm not sure if the providing resource might not require a reload to be removed in the future

Reply:
Ok, thanks. But, I just noticed that the name of file has a typo in it -- extra 'r'. s/listerner/listener

Reply:
ups, sorry, @bstansberry Done.